### DISCUSSION TITLE
add a 3rd temp storage

### DISCUSSION THREAD

**Discussion by Hardcore-fs** [madMAx43v3r/chia-plotter#949] [Ideas]
can we get a 3rd storage added in with auto copy.

so that when a plot is finished, you can roll it out of your 2 main drives into a 3rd and then from the 3rd to an external device.
currently if a plot is finished and you want to take it to NAS, it can take  20-60 mins due to the dual access on the temp folder.

by taking it to a local drive first you get it out of the data path of the main plotter.
Then a secondary copy can deal with this.

**Comment by altendky** [comment]
Sounds like maybe you want plotman's tmp/dst/archive setup.  Though for awhile I've mostly skipped dst since with madMAx I have plenty of left over tmp NVMe space since I purchased enough to work with chiapos.

**Comment by Hardcore-fs** [comment]
it is about path bandwidth........ not space...

**Reply by altendky** [reply]
Are the tmp disks spinners or SSD/NVMe?  None the less, your request seems at the very least similar to some of the plotman design considerations.  The dst layer was significantly intended as a buffer between the plot generation and archiving off to the plot's final resting place.  It specifically tries to avoid multiple plots being completed to or archived from a single dst drive so as to avoid the slow down you get when asking the disk to jump back and forth between two activities (seek times with spinners).

Yes, I maintain plotman, but I'm not looking to advertise.  Just trying to offer a maybe helpful suggestion.  I think that maintaining separation of responsibility between the core plotting code and the more general plot creation scheduling and plot archiving is good.
